rotkehlchen/db/history_events.py

```diff
@@ -356,6 +356,19 @@ def get_all_history_events(
     ) -> list[tuple[int, HistoryBaseEntry]]:
         ...
 
+    @overload
+    def get_all_history_events(
+            self,
+            cursor: 'DBCursor',
+            filter_query: HistoryBaseEntryFilterQuery,
+            has_premium: bool,
+            group_by_event_ids: bool,
+    ) -> Union[list[tuple[int, HistoryBaseEntry]], list[HistoryBaseEntry]]:
+        """
+        This fallback is needed due to
+        https://github.com/python/mypy/issues/6113#issuecomment-869828434
+        """
```

Comment:
Ah Isee you did it. Yes it's not. The reason is that mypy does not understand the variable is final. It can change at any point. Perhaps adding Final to the variable before putting it as an argument would work? But sitll probably not and this is a mypy thing to improve